hawkbit-rest/hawkbit-mgmt-resource/pom.xml

```diff
@@ -40,6 +40,7 @@
       <dependency>
          <groupId>org.springframework.plugin</groupId>
          <artifactId>spring-plugin-core</artifactId>
+         <version>${spring.plugin.core.version}</version>
```

Comment:
Is this really required? The plugin version is managed within hawkBit parent.

Reply:
Removed it.